apache-hbase-mixin/dashboards/apache-hbase-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1034 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'apache-hbase-cluster-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+
+local deadRegionServersPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'server_num_dead_region_servers{job=~"$job", hbase_cluster=~"$hbase_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{hbase_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Dead RegionServers',
+  description: 'Number of RegionServers that are currently dead.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'value',
+  },
+  pluginVersion: '10.2.0-61719',
+};
+
+local liveRegionServersPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'server_num_region_servers{job=~"$job", hbase_cluster=~"$hbase_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{hbase_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Live RegionServers',
+  description: 'Number of RegionServers that are currently live.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'value',
+  },
+  pluginVersion: '10.2.0-61719',
+};
+
+local regionserversPanel = {
```

Comment:
Do you think it makes sense to rework the "Is master" column? I had a very similar column in a table for Atlas, and it got worked to report the type rather than true or false and to be color coded. Without knowing enough, looking at this table, I imagine it'd be similar to renaming the column to be "Context" and the value is either "Master" or "Region server" and one of them has the text colored yellow and the other green. 

Just something to keep in mind that a similar change may be requested, unless grafana specified the column to be done the current way.

Reply:
That's a good idea, but I think 'context' is less obvious than 'Is master'. I'll look into some other options for naming that column. There are also a bunch of other changes to this table that I'm going to go over with Vitaly, so I'll bring that up with him at that point.